### PR TITLE
Add documentation for the FBS merchants api

### PR DESCRIPTION
**Pull request by Psilolouben**
Addresses [this task](https://phabricator.skroutz.gr/T112439)

### CODE REVIEW

**Review by tassosMaurides — COMMENTED** [review]
Purchase Orders descriptions looks fine, 
we should add documentation for fbs products too

File: source/localizable/smart_cart/_fbs_purchase_order_object.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#### Purchase Order object
+
+Name         | Type   | Description
+------------ | ------ | -----------------
+`code`       | String | Purchase Order code
+`received_at`| Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | Date the purchase order was received by the Skroutz warehouse
+`created_at` | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | Purchase Order creation date
+`expected_date` | Date String in format<br> `YYYY-MM-DD` | Date the purchase order is expected to be received by the Skroutz warehouse
```
Comment (tassosMaurides) [inline source/localizable/smart_cart/_fbs_purchase_order_object.html.md.erb:8]: same, omit `the`

**Review by tassosMaurides — COMMENTED** [review]
Overall, great job.
Just a comment, let's use `Skroutz warehouses` instead `the Skroutz warehouse` in all documentation

**Comment by Psilolouben** [comment]
> Overall, great job. Just a comment, let's use `Skroutz warehouses` instead `the Skroutz warehouse` in all documentation

Changed all occurencies to plurals.

**Review by tassosMaurides — APPROVED** [review]
:+1: Nice

**Pull request merged**